src/Compilers/Core/Portable/PEWriter/MetadataWriter.PortablePdb.cs

```diff
@@ -525,6 +535,15 @@ private static ImmutableArray<byte> SerializeBitVector(ImmutableArray<TypedConst
             return builder.ToImmutableAndFree();
         }
 
+        private static void SerializeTupleElementNames(BlobBuilder builder, ImmutableArray<TypedConstant> names)
+        {
+            builder.WriteInt32(names.Length);
+            foreach (var name in names)
+            {
+                builder.WriteSerializedString((string)name.Value);
```

Comment:
I'd use WriteUTF8(...) followed by WriteByte(0) instead (and writing just NUL -- that is, an empty string, if the name is missing).